design-system-templates/components/footer.html.twig

```diff
@@ -17,7 +17,6 @@
 				</li>
 			</ul>
 		</div>
-		<p class="copyright">Copyright © 20XX W3C <sup>®</sup> (<a href="https://www.csail.mit.edu/"><abbr title="Massachusetts Institute of Technology">MIT</abbr></a>, <a href="https://www.ercim.eu/"><abbr title="European Research Consortium for Informatics and Mathematics">ERCIM</abbr></a>,
-			<a href="https://www.keio.ac.jp/">Keio</a>, <a href="https://ev.buaa.edu.cn/">Beihang</a>) <a href="/Consortium/Legal/ipr-notice">Usage policies apply</a>.</p>
+		<p class="copyright">Copyright © 20XX World Wide Web Consortium. <a href="/Consortium/Legal/ipr-notice">Usage policies apply</a>.</p>
```

Comment:
There was no period before "Usage" but it's not clear to me whether this was a mistake or not

Reply:
As answered in the other MR, I added a period all the time, in particular because the next string "Usage policies apply" was already starting with an upper case.